docs/datasets.md

```diff
@@ -207,6 +207,19 @@ Offers an option to delete a dataset and completely remove it from webKnossos. B
 
 ![Dataset Editing: Delete Tab](images/delete_tab.jpeg)
 
+## Dataset Organization
+
+In the dashboard, you can see all datasets of your organization (if you have the necessary permissions).
+Datasets can be organized within folders by simply dragging a dataset entry from the table to a folder in the left sidebar.
+New folders can be created by right-clicking an existing folder in the sidebar and selecting "New Folder".
+
+A folder can be edited (also via the context menu) to change its name or its access permissions.
+In the access permissions field, a list of teams can be provided that controls which teams should have access to that folder.
+Note that even a folder with an empty access permissions field can be accessed by users which have access to its parent folder.
+This is because the access permissions are handled cumulatively.
```

Comment:
@fm3 does this explanation make sense to you? I found it hard to explain the concept in a simple way and I'm also not sure whether it's correct :see_no_evil:

Reply:
Sure, works for me :)